Form/Extension/CKEditorExtension.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Form\Extension;
+
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
+use Symfony\Cmf\Bundle\ContentBundle\Form\Type\StaticContentType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class CKEditorExtension extends AbstractTypeExtension
```

Comment:
lets consistently use the Ivory prefix (for the class, and also for the service xml file). the extension is specific to the ivory thing, not general ckeditor...